.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+name: release
+
+on:
```

Comment:
Linking conversation about the appropiate trigger here: https://github.com/elastic/ecs-dotnet/commit/b5e88ff2d9dbb495368326028ad4f4ee91ea02e4